### ISSUE TITLE
rasterio 1.3

### ISSUE THREAD

**Issue by emanuel-schmid** [CLIMADA-project/climada_python#512]
### rasterio.crs.CRS
This is not a member anymore

### changed way to deal with GMT_intermediate_coast_distance_01d.tif
The unit test `climada.util.test.test_coordinates.TestGetGeodata.test_dist_to_coast_nasa` fails with rasterio=1.3.0

**Error Message**

> 416.66666667 != 515.7736607600339 within 7 places (99.10699409003388 difference)

**Stacktrace**
```
Traceback (most recent call last):
  File "/var/lib/jenkins/jobs/climada_branches/branches/develop/workspace/climada/util/test/test_coordinates.py", line 948, in test_dist_to_coast_nasa
    self.assertAlmostEqual(d, r)
AssertionError: 416.66666667 != 515.7736607600339 within 7 places (99.10699409003388 difference)
```

with rasterio=1.2.10, the test passes.

This can be reproduced
```
import numpy as np
from climada.util import read_raster_sample
path = '~/climada/data/GMT_intermediate_coast_distance_01d.tif'
lat, lon = [13.20833333, -12.497529], [-59.625, -58.849505]
print(read_raster_sample(path, np.array(lat), np.array(lon), intermediate_res=0.1))
```
up to rasterio 1.2.10, this used to print `416.666`, with 1.3.10 `515.773`

**Comment by emanuel-schmid** [comment]
More failing tests that have been fixed by downgrading to rasterio 1.2.10:
- [climada.test.test_litpop_integr.TestLitPopExposure.test_netherlands150_pass](https://ied-wcr-jenkins.ethz.ch/job/climada_ci_night/147/testReport/climada.test.test_litpop_integr/TestLitPopExposure/test_netherlands150_pass/)
- [climada.test.test_litpop_integr.TestAdmin1.test_brandenburg](https://ied-wcr-jenkins.ethz.ch/job/climada_ci_night/147/testReport/climada.test.test_litpop_integr/TestAdmin1/test_brandenburg/)

**Comment by peanutfun** [comment]
Resolved, apparently there was an underlying issue with the initial rasterio v1.3 release.
